docs/topics/mta-web-tagging-an-application.adoc

```diff
@@ -58,4 +58,56 @@ A side drawer opens.
 +
 You can see the tags attached to the application.
 +
-The tags can be filtered by source and tag category. The sources are *Analysis* and *Manual*. The categories are displayed in a drop-down list, for example, *HTTP*, *MVC*, *Web*, *Observability*, *Persistence*, *Application Type*, *Data Center*.
\ No newline at end of file
+The tags can be filtered by source and tag category. The sources are *Analysis* and *Manual*. The categories are displayed in a drop-down list, for example, *HTTP*, *MVC*, *Web*, *Observability*, *Persistence*, *Application Type*, *Data Center*.
+
+[id="creating-tags_{context}"]
+== Creating tags
+
+You can create custom tags for applications for applications that {ProductShortName} assesses or analyzes.
+
+.Procedure
+
+. In *Migration* view, click *Controls*.
+. Click the *Tags* tab.
+. Click *Create tag*, which opens a dialog.
+. In the *Name* field enter a unique name for the tag.
+. Click the *Tag category* field and select the category tag to associate with the field.
+. Click *Create*.
+
+[id="editing-tags_{context}"]
+= Editing tags
+
+You can edit defined tags.
+
+.Procedure
+
+. In *Migration* view, click *Controls*.
+. Click the *Tags* tab. A list of tag categories appears in the main pane.
+. Click the arrow to the left of the category name, opening a list of tags in the category.
+
+. Select a defined tag category and click the arrow to the left of the category name, opening a list of tags.
+. Select *Edit* from the drop-down menu, opening a dialog.
+. Edit the tag's name in the *Name* field.
+. Click the *Tag category* field and and select the category tag to associate with the field.
+. Click *Save*.
+
+
+[id="editing-tag-categories_{context}"]
+= Editing tag categories
+
+You can edit defined tag categories.
+
+.Procedure
+
+. In *Migration* view, click *Controls*.
+. Click the *Tags* tab.
++
+
+. Select a defined tag category and click *Edit* to the right of the tag category.
+. In the *Name* field enter a unique name for the tag category, opening a dialog.
```

Comment:
- "opening a dialog." is not relevant here, but to the previous step
- this step is duplicated, please remove this line